recipe/meta.yaml

```
@@ -79,19 +68,30 @@ test:
     - skbio.tree.tests
     - skbio.util
     - skbio.util.tests
+  requires:
+    - pip
+    - pytest
+    - responses
+    - matplotlib
+    - biom-format
+    - array-api-compat
+  script_env:
+    - MPLBACKEND=Agg
   commands:
-    - MPLBACKEND='Agg' python -m skbio.test
+    - pip check
+    - python -c "from importlib.metadata import version as v; assert v('scikit-bio')=='{{ version }}'"
+    - python -m skbio.test
 
 about:
-  home: http://scikit-bio.org
+  home: https://scikit-bio.org
```

Actually https://github.com/scikit-bio/scikit-bio/blob/0.7.0/pyproject.toml#L51

```suggestion
  home: https://scikit.bio
```